server/src/main/java/com/angelomelonas/grpcwebchat/subscription/SubscriptionData.java

```diff
@@ -0,0 +1,32 @@
+package com.angelomelonas.grpcwebchat.subscription;
+
+import com.google.protobuf.Any;
+
+import java.util.UUID;
+
+public class SubscriptionData {
```

Comment:
I think this class was created too soon ... first demonstrate why it must exist in your unit tests